doc/user-guide/16-Rubrik-CDM.adoc

```diff
@@ -0,0 +1,81 @@
+= Documentation for the Rubrik (CDM) Method
+
```

Comment:
@DamaniN 
I know absolutely nothing about what `the Rubrik (CDM) Method` could be
because I nothing at all to do with `Rubrik Cloud Data Management`.
So I would appreciate some initial abstract what that is all about.
You should also tell here what that `CDM` abbreviation stands for
(except you want to scare away right from the start all readers who do not know what that is ;-)